# Custom message for ducklings package (ref from turtlesim/msg/Pose.msg) 
# Intended to be used to  
Header header

float32 x
float32 y
float32 z
float32 theta

float32 linear_velocity
float32 angular_velocity
